vcpkg/triplets/arm-neon-android.cmake

```diff
@@ -0,0 +1,9 @@
+set(VCPKG_TARGET_ARCHITECTURE arm)
+set(VCPKG_CRT_LINKAGE dynamic)
+set(VCPKG_LIBRARY_LINKAGE static)
+set(VCPKG_CMAKE_SYSTEM_NAME Android)
+set(VCPKG_BUILD_TYPE release)
+set(VCPKG_MAKE_BUILD_TRIPLET "--host=armv7a-linux-androideabi")

```

Comment:
Neon is back? I don't see any different between this and arm-android triblet, and I missing something?

Reply:
It's used (by triplet name) here:
https://github.com/microsoft/vcpkg/blob/master/scripts/toolchains/android.cmake#L17

We should only be using that one and we will need to when we switch to a newer ndk.

Reply:
@m-kuhn , K, makes sense.